src/codeflare_sdk/cluster/cluster.py

```diff
@@ -61,6 +69,26 @@ def __init__(self, config: ClusterConfiguration):
         self.config = config
         self.app_wrapper_yaml = self.create_app_wrapper()
         self.app_wrapper_name = self.app_wrapper_yaml.split(".")[0]
+        self._client = None
+        self._k8_client = config.new_client_from_config()
```

Comment:
I think config is getting overloaded here. I just got the error: `AttributeError: 'ClusterConfiguration' object has no attribute 'new_client_from_config'`. 

It seems like you want to use `kubernetes.config` but just above you are setting `self.config` to the ClusterConfiguration object.

Reply:
Thanks, nice catch